gooddata-pandas/gooddata_pandas/result_convertor.py

```diff
@@ -197,19 +197,27 @@ def _read_complete_execution_result(
 
 
 def _create_header_mapper(
-    response: BareExecutionResponse, dim: int, label_overrides: Optional[LabelOverrides] = None
+    response: BareExecutionResponse,
+    dim: int,
+    label_overrides: Optional[LabelOverrides] = None,
+    use_local_ids_in_headers: Optional[bool] = None,
 ) -> Callable[[Any, Optional[int]], str]:
     """
     Prepares header mapper function which is doing header structures translations into appropriate label used
     in a dataframe
     :param response: Response structure to gather dimension header details
     :param dim: dimension id
     :param label_overrides: label overrides
+    :param use_local_ids_in_headers: Use local identifiers of header attributes and metrics. Optional.
+
     :return: Mapper function
     """
     if label_overrides is None:
         label_overrides = {}
 
+    if use_local_ids_in_headers is None:
+        use_local_ids_in_headers = False
```

Comment:
Why do not have `use_local_ids_in_headers` set to False? Then you will not need this code.

Reply:
fixed